site/content/en/docs/adoption/_index.md

```diff
@@ -39,9 +39,14 @@ aliases:
 
 [**vLLM**](https://github.com/vllm-project/vllm): vLLM is a fast and easy-to-use library for LLM inference, it can be deployed with LWS on Kubernetes for distributed model serving, see documentation [here](https://docs.vllm.ai/en/stable/deployment/frameworks/lws.html).
 
-[**sglang**](https://github.com/sgl-project/sglang): sglang, a fast serving framework for large language models and vision language models. It can be deployed with LWS on Kubernetes for
+[**SGLang**](https://github.com/sgl-project/sglang): SGLang, a fast serving framework for large language models and vision language models. It can be deployed with LWS on Kubernetes for
 distributed model serving, see documentation [here](https://docs.sglang.ai/references/deploy_on_k8s.html#deploy-on-kubernetes)
 
+[**OME**](https://github.com/sgl-project/ome): OME is a Kubernetes operator for enterprise-grade management and serving of LLMs.
```

Comment:
Can we use this time to alphabetize the list? Thanks!

Reply:
done. Thanks